source/libs/cache.ts

```diff
@@ -59,8 +57,8 @@ async function purge(): Promise<void> {
 }
 
 // Automatically clear cache every day
-if (location.pathname === '/_generated_background_page.html') {
+if (isBackgroundPage()) {
 	// Call immediately to ensure the cache gets purged at some point.
```

Comment:
I preferred my comment, it's no longer "immediate" https://github.com/sindresorhus/refined-github/pull/2109#discussion_r290444434

Reply:
Sorry I missed that one.